### ISSUE TITLE
Multiple Language

### ISSUE THREAD

**Issue by 1337god** [witch-house/pronoun.is#124]
Ability to have a multiple language and allow people to provide translation.
for example https://pronoun.is/pt/he for portuguese page.
Maybe using Weblate for the translations?

**Comment by kalikiana** [comment]
I think this is going to be a bit more difficult. And here's why:

English: they/their/themself
Spanish: elle
German: xier/xier/xiem/xieser/xien/xieses/xiesa/xiesem

And let's just say this is what I choose for myself. That doesn't mean everyone who uses they/themself in English will use xier in German. Perhaps they prefer German *ersie* and Spanish *ell@*?

**Comment by patch** [comment]
I agree with @kalikiana. Many language pairs don't have clear 1-to-1 pronoun translations (even among traditional gendered pronouns) and also pronouns are a personal choice in each language.
